docs/guides/api-gateway/get-started.mdx

```diff
@@ -0,0 +1,641 @@
+      "card_brand": "VISA",
+      "card_type": "CREDIT",
+      "cardholder_name": "Amelia Earhart",
+      "customer_id": "VDKXEEKPJN48QDG3BGGFAK05P8",
+      "enabled": true,
+      "exp_month": 11,
+      "exp_year": 2018,
+      "last_4": "1111",
+      "prepaid_type": "NOT_PREPAID",
+      "reference_id": "user-id-1",
+      "version": 1
+    }
+  ]
+}
+```
+
+Stop the agent to remove the internal endpoint you just created.
+
+## Manage APIs with Traffic Policy rules
+
+Your API is now online, but it is completely exposed to the outside world. Next, add some traffic policy rules to authenticate
+requests and shape the traffic that reaches your backend service. You'll do this by running a command to update the configuration
+of your existing cloud endpoint, so you'll need the `id` from the previous step.
+
+If you didn't save the id when you created
+the cloud endpoint, run the following command, locate your endpoint, and grab its `id`:
+
+```
+ngrok api endpoints list
+```
+
+### Route by headers
+
+You can configure traffic policy rules to direct traffic based on HTTP header values.
+
+#### Start internal endpoints
+
+To implement this example, run each of the following commands in a separate terminal to start internal endpoints to the two instances of the sample API:
+
+```bash
+ngrok http 8001 --url=https://v1.api.internal
+ngrok http 8002 --url=https://v2.api.internal
+```
+
+While these endpoints point to two separate instances of the same backend API, they simulate the scenario where you need to direct traffic to two
+separate API services depending on the version.
+
+#### Update your policy rules
+
+This example looks for a header named `X-Api-Version` and routes traffic to different instances of the `/cards` route of the sample API.
+Feel free to modify it for your own API if you're not using the sample app. Update your `policies.yaml` file with the contents below,
+overwriting all previous contents:
+
+```yaml
+---
+on_http_request:
+  - name: "Route traffic to API v1"
+    expressions:
+      - "'1' in req.headers['x-api-version']"
+    actions:
+      - type: forward-internal
+        config:
+          url: https://v1.api.internal
+          binding: internal
+  - name: "Route traffic to API v2"
+    expressions:
+      - "'2' in req.headers['x-api-version']"
+    actions:
+      - type: forward-internal
+        config:
+          url: https://v2.api.internal
+          binding: internal
+  - actions:
+      - type: forward-internal
+        config:
+          url: https://v2.api.internal
+          binding: internal
+```
+
+:::note
+You must always include the `forward-internal` action as the final step in your policy file since once it executes,
+no subsequent policy rules will be executed. Additionally, you always need a terminating policy rule.
+In this case, the traffic forwards to `https://v2.api.internal` if the header is set to a value other than `1` or `2` or
+if the header isn't present in the request.
+:::
```

Comment:
Here and in the other notes about having to end with a `forward-internal` action is not *quite* accurate. 

The traffic policy validation (for cloud endpoints) forces you to have some action implemented that is guaranteed to do _something_ with traffic on your endpoints. This could be `forward-internal` but this could also be `deny` or `custom-response`.

Reply:
That's totally true! I'll update these warnings to be more technically accurate. Thanks for the feedback, Rachel.